utils/release/npm-publish-package.mjs

```diff
@@ -1,222 +1,24 @@
 #!/usr/bin/env node
```

Comment:
This is the second half of the former `npm-publish-package.mjs`. Essentially, nothing changed.